src/parser/druid/feral/CHANGELOG.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 
-import { Abelito75, Zeboot, LeoZhekov, Tora, Xcessiv } from 'CONTRIBUTORS';
+import { Abelito75, Zeboot, LeoZhekov, Tora, Xcessiv, Tiboonn } from 'CONTRIBUTORS';
 import { change, date } from 'common/changelog';
 
 export default [
+  change(date(2021, 1, 16), <>Add class conduits</>, Tiboonn),
```

Comment:
I would remove the fragments from these  (The <></>) since it is just text.

And to be clear, i would change it to "Added spell information for conduits", cause adding the spell IDs and icons and such to SPELLS doesnt actually add the conduits to anything, it just allows them to be added to stuff

Reply:
Good feedback! Thanks 
I already noticed the React error, fixed that.
Also updated the change log message.

Wanted to start with something simple to get familiar with the project and the code, to maybe later update the Balance druid class :)